daemon/api_system_volumes.go

```diff
@@ -141,7 +142,7 @@ func postSystemVolumesActionReplaceRecoveryKey(c *Command, req *systemVolumesAct
 		return BadRequest("cannot replace recovery key: %v", err)
 	}
 
-	chg := st.NewChange("replace-recovery-key", "Replace recovery key")
+	chg := newChange(st, "fde-replace-recovery-key", "Replace recovery key", []*state.TaskSet{ts}, nil)
 	chg.AddAll(ts)
```

Comment:
we don't need this anymore, if you pass it above

Reply:
thanks for catching this, my bad. updated the unit test to catch this bug.